src/bench.jl

```diff
@@ -55,9 +51,15 @@ function bench()
     # open(filename, "w") do io
       benchmark_bw_cu(io, model(), n)
     # end
-  
+
     # open(filename, "w") do io
       benchmark_cu(io, model(), n)
     # end
   end
+
+  for model in [ObjectDetector.YOLO.v3_608_COCO, ObjectDetector.v3_tiny_416_COCO]
+    for batchsize in [1, 3]
```

Comment:
Do we want to try other models? I also think having bigger batch sizes would make sense right?

Reply:
Thanks!